forge-test/deployment/ChangeOwnerToMultiSig.t.sol

```diff
@@ -0,0 +1,158 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.25;
+
+import "lib/forge-std/src/Test.sol";
+import "lib/forge-std/src/console.sol";
+import {ChangeOwnerToMultiSig} from "../../forge-scripts/deployment/ChangeOwnerToMultiSig.s.sol";
+import {HelperConfig} from "../../forge-scripts/HelperConfig.s.sol";
+import {LiquidityBridgeContract} from "../../contracts/legacy/LiquidityBridgeContract.sol";
+import {LiquidityBridgeContractProxy} from "../../contracts/legacy/LiquidityBridgeContractProxy.sol";
+import {LiquidityBridgeContractAdmin} from "../../contracts/legacy/LiquidityBridgeContractAdmin.sol";
+
+/**
+ * @title ChangeOwnerToMultiSigTest
+ * @notice Test for the ChangeOwnerToMultiSig deployment script
+ * @dev Tests ownership transfer pattern to multisig
+ */
+contract ChangeOwnerToMultiSigTest is Test {
```

Comment:
I think the original tests validates that the new owner can perform only owner operations